public/index.html

```diff
@@ -1,18 +1,14 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en-US">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <!-- <title><%= htmlWebpackPlugin.options.title %></title> -->
```

Comment:
I wonder if this WebpackPlugin thing is actually needed for compiling or if it was ok to delete it 🤔 Guess we'll find out!